bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/RawType.java

```diff
@@ -22,32 +22,45 @@
  */
 public class RawType implements PrimitiveType, State {
 
+    private static final String DEFAULT_MIME_TYPE = "image/jpeg";

```

Comment:
We could create a `ImageType` class and keep `RawType` unchanged if you prefer ?
But it will require an update in all (only 5) bindings using RawType for image.